subsys/tracing/sysview/Kconfig

```diff
@@ -12,6 +12,16 @@ config SEGGER_SYSVIEW_RTT_BUFFER_SIZE
 	depends on SEGGER_SYSTEMVIEW
 	default 4096
 
+config SEGGER_SYSVIEW_RTT_CHANNEL
+	int "RTT channel for SystemView"
+	depends on SEGGER_SYSTEMVIEW
+	default 0
+
+config SEGGER_SYSVIEW_APP_NAME
+	string "Application name to be displayed in SystemView"
+	depends on SEGGER_SYSTEMVIEW
+	default "ZephyrSysView"
```

Comment:
I am quite new to working on Zephyr. We should be able to use Kconfig values in cmake but I am not sure the other way around is possible. If anyone has an idea we could add it. Otherwise it may be worth creating an issue asking if accessing cmake variables from Kconfig is possible at all.